apis/ocpdiag/ocpdiag/build_deps.bzl

```diff
@@ -159,3 +156,11 @@ def load_deps(ocpdiag_package_name = "ocpdiag"):
             branch = "master",
             remote = "https://github.com/google/ecclesia-machine-management.git",
```

Comment:
we don't need those yet

Reply:
That's true, but it would be a bit annoying to split deps for this and GoB. My preference would be to just leave it (same for below comment)

Reply:
SGTM